include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -40,23 +40,46 @@ namespace NovelRT::Maths
         float x;
         float y;
 
+        /**
+         * @brief Construct new GeoVector2F with X and Y components set to 0
+         * @returns Newly constructed GeoVector2F
+         */
         GeoVector2F() noexcept : x(0.0f), y(0.0f)
         {
         }
+
+        /**
+         * @brief Construct new GeoVector2F with X and Y components with x and y paramters
+         * @param x Set X Component Value
+         * @param y Set Y Component Value
+         * @returns Newly constructed GeoVector2F
+         */
```

Comment:
Minor tweaks from me.
```suggestion
        /**
         * @brief Constructs a new GeoVector2F the provided X and Y values.
         * @param x The X Component Value.
         * @param y The Y Component Value.
         * @returns Newly instantiated GeoVector2F.
         */
```